graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -560,7 +561,9 @@ def _compile_vertex_ast(schema, current_schema_type, ast,
     return basic_blocks
 
 
-def _validate_fold_has_outputs_or_count_filter(fold_scope_location, fold_has_count_filter, outputs):
+def _validate_fold_has_outputs_or_count_filter(
+    fold_scope_location, fold_has_count_filter, metadata
```

Comment:
Let's always try to refer to the metadata table object as `query_metadata_table` , for consistency.